lib/data/wrapper.js

```diff
@@ -153,6 +153,25 @@ const data = {
         client = newClient;
         return client;
     },
+
+    checkHealth: (log, cb) => {
+        client.healthcheck(log, (err, result) => {
+            const respBody = {};
+            if (err) {
+                log.error('error from sproxyd', { error: err });
```

Comment:
I like how the response is generic as to implementation.  Let's make the log that way too.  So, instead of 'error from sproxyd', it should be error from implName.